frontend/src/metabase/lib/settings.js

```diff
@@ -119,6 +124,19 @@ class Settings {
     return this.get("anon-tracking-enabled") || false;
   }
 
+  versionInfoLastChecked() {
+    const ts = this.get("version-info-last-checked");
+    if (ts) {
+      // app DB stores this timestamp in UTC, so convert it to the local zone to render
+      return moment
+        .utc(parseTimestamp(ts))
+        .local()
+        .format("MMMM Do YYYY, h:mm:ss a");
+    } else {
+      return "never";
```

Comment:
I think this `"never"` string needs to be translated as well

Reply:
Like this?